docs/how_to_dev_frontend_en.md

```diff
@@ -0,0 +1,105 @@
+# VisualDL Web App Develop
+
+VisualDL has two major components. One is the Python/C++ SDK that logs the matrix during training,
+the other is the web-app that display visualized training data.
+
+The web-app reads the extract data and draw scalars graph or histograms graphs.
+The web-app is where VisualDL transforms the code data into graphs so users can visualize the training progress and make a better network design evaluation.
```

Comment:
`code data`?

Reply:
`network design evaluation`? is that a thing?

Reply:
I will remove `code`

Don't we use VisualDL to evaluate the Neural Network?

Reply:
So here the word "graph" is the tricky one. For many people, the first thing that might come to their mind is the computation graph and not the 2-D graph, as you intend to use it. That is why, to remove ambiguity, they use "scalar" plotting / graphing. You also would need to use that.

And I think "network architecture" is a more unambiguous term than "network design" - even though in all fairness you are right and I get what you are trying to say.